TeensySF/config.ini

```diff
@@ -1 +1,99 @@
+# starts to fade in. Default 100
+humStart=600
+
+# The volume of the hum sound. Can be 0-16, where 0 is muted.
+# Default 15
+volHum=15
+
+# The volume of all effect sounds. Can be 0-16, where 0 is muted.
+# Default 16
+volEff=16
+
+# How fast (degrees per second) we have to swing before a swing effect is
+# triggered. Default 250.
+ProffieOSSwingSpeedThreshold=250
+
+# How much to bend the response curve between swing speed and swing volume.
+# Can be 0.01 - 2.0, where value of 1.0 will result in no bending. Default 0.5
+ProffieOSSwingVolumeSharpness=0.5
+
+# The volume multiplier when swings are at the swing speed threshold.
+# Can be 1.0 to 3.0. Default 2.0
+ProffieOSMaxSwingVolume=2.0
+
+# Specify what fraction of swing that must be played before a new swing can be
+# started. Can be 0.0-1.0. Defaults to 0.5 (50%).
+ProffieOSSwingOverlap=0.5
+
+# How much to duck the hum when the swing is playing.                        
+# Defaults to 0.2 (hum volume is decreased by 20% of swing volume)      
+ProffieOSSmoothSwingDucking=0.2
+
+# How slow (degrees per second) the swing has to be before it's not considered a
+# swing anymore. Default 200.
+ProffieOSSwingLowerThreshold=200
+
+# Only used for non-smoothswing fonts. Specifies how aggressive a swing has to be to be considered a slash. Once we
+# reach the ProffieOSSwingSpeedThreshold, rate of swing speed change is used to
+# determine if it's a swing or a slash. Default 260
+ProffieOSSlashAccelerationThreshold=260
+
+# For OLED displays. This specifies the frame rate of animations.
+# If not specified (or set to zero) the frame rate will be 24 frames
+# per second for non-looped animations. For looped animations, the
+# frame rate will be set so that the loop takes one second.
+ProffieOSAnimationFrameRate=0.0
+
+# For OLED displays, the time a blade style's "message", 
+# static BMP, or loop animation will play when changing presets.
+ProffieOSFontImageDuration=5000.0
+
+# For OLED displays, the time an on.bmp will play
+ProffieOSOnImageDuration=5000.0
+
+# For OLED displays, the time a blst.bmp will play
+ProffieOSBlastImageDuration=1000.0
+
+# For OLED displays, the time a clsh.bmp will play
+ProffieOSClashImageDuration=500.0
+
+# For OLED displays, the time a force.bmp will play
+ProffieOSForceImageDuration=1000.0
+
+# If #define ENABLE_SPINS is defined. Number of degrees the blade must travel while staying above the
+# swing threshold in order to trigger a spin sound.  Default is 360 or
+# one full rotation.
+ProffieOSSpinDegrees=360.0
+
+### ---- These below as of OS6 only ---- ###
+
+# Minimum acceleration for Accent Swing file Selection
+#recommended value is 20.0 ~ 30.0
+ProffieOSMinSwingAcceleration=20.0
+
+# Maximum acceleration for Accent Swing file Selection
+#must be higher than Min value to enable selection
+#recommended value is 100.0 ~ 150.0
+ProffieOSMaxSwingAcceleration=100.0
+
+# Match sequential effects such as preon->out.wav files
```

Comment:
This is all hypothetical for this font, and I think this information belongs on the wiki rather than in this file. If anything I think it's just going to confuse people.

It's fine to have explanations for this in a font that actually uses it though.

Reply:
Tailored to be appropriate for this font.